examples/placeholder/linux/apps/app1/zap_upgrades.yaml

```diff
@@ -0,0 +1,5 @@
+upgrade_results:
```

Comment:
What do these do?

Reply:
This is indicating that there was a change in the SDK which led to a zap configuration change.

Reply:
This is based off the design mentioned here https://github.com/project-chip/zap/blob/master/docs/zap-file-upgrade.md .
It is a way to help users upgrade their zap files appropriately when they update their existing apps to a new Matter SDK.